packages/components/psammead-section-divider/src/index.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import styled from 'styled-components';
+import { boolean, shape, string } from 'prop-types';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import { MEDIA_QUERY_TYPOGRAPHY } from '@bbc/gel-foundations/breakpoints';
+import { GEL_SPACING } from '@bbc/gel-foundations/spacings';
+import {
+  getDoublePica,
+  GEL_FF_REITH_SANS,
+} from '@bbc/gel-foundations/typography';
+import { C_CLOUD_LIGHT, C_EBON, C_WHITE } from '@bbc/psammead-styles/colours';
+
+const TOP_MARGIN_PX = 1;
+
+const calcTop = ({ inline, script }) =>
+  inline
+    ? `
+      // place at middle of text line height
+      top: ${(script.doublePica.groupA.lineHeight / 2 + TOP_MARGIN_PX) / 16}rem;
+
+      ${MEDIA_QUERY_TYPOGRAPHY.SMART_PHONE_ONLY} {
+        top: ${(script.doublePica.groupB.lineHeight / 2 + TOP_MARGIN_PX) /
+          16}rem;
+      }
+
+      ${MEDIA_QUERY_TYPOGRAPHY.LAPTOP_AND_LARGER} {
+        top: ${(script.doublePica.groupD.lineHeight / 2 + TOP_MARGIN_PX) /
+          16}rem;
+      }
+    `
+    : 'top: 0;';
+
+const SectionDividerWrapper = styled.div`
+  position: relative;
+  &::before {
+    content: '';
+    position: absolute;
+    height: 1px;
+    border: none;
+    background: ${C_CLOUD_LIGHT};
+    left: 0;
+    right: 0;
+    ${calcTop};
+  }
+`;
+
+const SectionTitle = styled.h2`
+  ${props => getDoublePica(props.script)};
+  color: ${C_EBON};
+  background-color: ${C_WHITE};
+  font-family: ${GEL_FF_REITH_SANS};
+  display: inline-block;
+  position: relative;
+  margin: ${TOP_MARGIN_PX}px 0 0 0;
```

Comment:
This can probably just be this right? 
```suggestion
  margin-top: ${TOP_MARGIN_PX}px;
```

Reply:
Can it? My thinking was to force the default margins to 0 so that margins added to the Wrapper or through the container can control the placement

Reply:
Ah good point